consensus/src/network_tests.rs

```diff
@@ -315,6 +315,16 @@ impl NetworkPlayground {
             .unwrap()
             .stop_drop_message_for(src, dst)
     }
+
+    pub async fn start<T: Payload>(mut self) {
```

Comment:
Do you remember why we didn't do it before? Looks like a very natural thing to have, did we just miss it? :)

Reply:
back in old days, we rely on `playground.wait_for` to inspect messages both in event_processor_tests and chained_bft_smr tests